### PR TITLE
Wagtail 5.1

### PR DESCRIPTION
**Pull request by katdom13**
## What's new:
- Add tests for Wagtail 5.1

**Upstream PR**: https://github.com/springload/wagtail-django-recaptcha/pull/47

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -7,8 +7,8 @@
 skipsdist = True
 usedevelop = True
 envlist =
-    py{38,39,310}-dj{32,41}-wt{41,42,50}
-    py{311}-dj{41,42}-wt{50}
+    py{38,39,310,311}-dj{32,41}-wt{41,42,50}
+    py{39,310,311}-dj{42}-wt{51}
```
Comment (nickmoreton) [inline tox.ini]: Seeing that dj32 isn't supported on py311 can the matrix be adjusted please.

File: .travis.yml [hunk 2]
```diff
@@ -31,20 +31,34 @@ matrix:
...
+   - env: TOXENV=py311-dj32-wt42
      python: 3.11
+   - env: TOXENV=py311-dj32-wt50
+     python: 3.11
+   - env: TOXENV=py310-dj41-wt41
+     python: 3.10
+   - env: TOXENV=py310-dj41-wt42
+     python: 3.10
+   - env: TOXENV=py310-dj41-wt50
+     python: 3.10
```
Comment (nickmoreton) [inline .travis.yml]: Should there be an entry for wt51 in this file?

**Review by nickmoreton — CHANGES_REQUESTED** [review]
Hi @katdom13 I left a couple of change requests.

File: tox.ini (hunk 1 above)
Comment (katdom13) [reply tox.ini]: How about this?
https://github.com/torchbox-forks/wagtail-django-recaptcha/pull/8/commits/5282c47a8b2a08fc91909ea8e0d265c1f67ee0a2

File: .travis.yml (hunk 2 above)
Comment (katdom13) [reply .travis.yml]: Good eye! I completely forgot. Please see this commit:
https://github.com/torchbox-forks/wagtail-django-recaptcha/pull/8/commits/3fcdf14c298560abc056949beef687b0fcd2c14c

**Review by nickmoreton — APPROVED** [review]
Thanks @kat all looks good.

**Pull request merged**